src/common/schemas/input/sign.json

```diff
@@ -35,6 +35,10 @@
         "signAs": {
           "$ref": "address",
           "description": "The account that the signature should count for in multisigning."
+        },
+        "setFullyCanonicalSig": {
+          "type": "boolean",
+          "description": "If `false`, the sign method will not set the tfFullyCanonicalSig flag on the transaction. The default is `true`. [New in ripple-lib {VERSION}.]"
```

Comment:
Is `{VERSION}` going to get changed when this PR is released? Maybe it should be marked with a TODO or something just to make that clear.

Reply:
Yes.